### PR TITLE
Fix LLVM 10 build

### PR DESCRIPTION
**Pull request by mailwl**

### CODE REVIEW

File: GPCS4/Common/BitHelper.h [hunk 1]
```diff
@@ -29,7 +29,9 @@ namespace bit {
     return n;
   }
 
+#ifndef _tzcnt_u32
```
Comment (Inori) [inline GPCS4/Common/BitHelper.h]: Why use `#ifdef`? It seems `_tzcnt_u32` is a normal function declaration, not macro.

File: GPCS4/Graphic/Pssl/GCNParser/ParserSI.h [hunk 2]
```diff
@@ -59,50 +59,50 @@ class ParserSI
...
+        InstructionEncoding_SOP1 = 0x0000017Du << 23,
         /// bits [31:23] - (1 0 1 1 1 1 1 1 1)
-        InstructionEncoding_SOPP = 0x0000017F << 23,
+        InstructionEncoding_SOPP = 0x0000017Fu << 23,
         /// bits [31:23] - (1 0 1 1 1 1 1 1 0)
-        InstructionEncoding_SOPC = 0x0000017E << 23,
+        InstructionEncoding_SOPC = 0x0000017Eu << 23,
         /// InstructionEncodingMask_7bit
         /// bits [31:25] - (0 1 1 1 1 1 1)
         InstructionEncoding_VOP1 = 0x0000003F << 25,
```
Comment (Inori) [inline GPCS4/Graphic/Pssl/GCNParser/ParserSI.h]: Keep it uniform, like other enum values.

File: GPCS4/Common/BitHelper.h (hunk 1 above)
Comment (mailwl) [reply GPCS4/Common/BitHelper.h]: it already defined in .h file, and if not add #ifdef clang does error:
GPCS4\GPCS4\Common\BitHelper.h(33,34): error : C++ requires a type specifier for all declarations
(don't know, how to fix it other way)

Comment (Inori) [reply GPCS4/Common/BitHelper.h]: Can you find `#define _tzcnt_u32` in clang 10 enverienment? I can only find normal declaration in clang 9 and 8.

Comment (mailwl) [reply GPCS4/Common/BitHelper.h]: no, it's not #defined, usual `extern` function

Comment (Inori) [reply GPCS4/Common/BitHelper.h]: OK, it seems this is a compatibility bug of clang-cl, search "clang _tzcnt_u32" on google will give many issue reports. It's ok if this approach can fix the problem. https://bugs.llvm.org/show_bug.cgi?id=25544

**Pull request merged**